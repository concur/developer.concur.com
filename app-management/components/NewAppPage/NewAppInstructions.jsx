import React from 'react';
import { Link } from 'react-router';

const NewAppInstructions = () => (
  <div>
    <h3>Welcome to App Management</h3>
    <p>
      Create a new app by going to the <Link to="/new">Create an App form</Link>
    </p>
  </div>
);

export default NewAppInstructions;
