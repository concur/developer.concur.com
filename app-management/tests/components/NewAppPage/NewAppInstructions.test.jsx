import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import NewAppInstructions from '../../../components/NewAppPage/NewAppInstructions';

describe('<NewAppInstructions />', () => {
  it('renders with instructions on making a new app', () => {
    const page = shallow(<NewAppInstructions />);

    expect(shallowToJson(page)).toMatchSnapshot();
  });
});
