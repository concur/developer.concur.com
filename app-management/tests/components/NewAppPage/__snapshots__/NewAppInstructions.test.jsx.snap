exports[`<NewAppInstructions /> renders with instructions on making a new app 1`] = `
<div>
  <h3>
    Welcome to App Management
  </h3>
  <p>
    Create a new app by going to the 
    <Link
      onlyActiveOnIndex={false}
      style={Object {}}
      to="/new">
      Create an App form
    </Link>
  </p>
</div>
`;
